crypto_cpp_py/utils.py

```diff
@@ -0,0 +1,43 @@
+import hashlib
```

Comment:
Looking good 👌 
Do you think it would be a good idea to add a comment at the top stating from which cairo-lang commit it was taken, for possible future referrence?